### ISSUE TITLE
Flashcard reviewer: long math formulas are clipped without horizontal scroll

### ISSUE THREAD

**Issue by MaolinSun32** [st3v3nmw/obsidian-spaced-repetition#1451]
## Bug Description

When a LaTeX formula is wider than the flashcard reviewer viewport, the formula is clipped on the right side and there is **no horizontal scrollbar** to view the rest of it.

In contrast, Obsidian's built-in reading view renders the same formula with a **gray horizontal scrollbar** at the bottom of the math block, allowing the user to scroll and see the complete expression.

## Steps to Reproduce

1. Create a flashcard containing a wide `$$...$$` block formula (e.g. a BatchNorm backprop equation with `\underbrace` annotations)
2. Open the flashcard reviewer
3. Observe that the right portion of the formula is cut off with no way to scroll

## Expected Behavior

The flashcard reviewer should provide a horizontal scrollbar (or `overflow-x: auto` on the MathJax/KaTeX container), matching Obsidian's native reading-view behavior.

## Screenshots

**Flashcard reviewer** — formula clipped, no scrollbar:
<img width="1006" height="574" alt="Image" src="https://github.com/user-attachments/assets/bf5368f8-2d66-4b2e-8704-221d40c497ce" />

**Obsidian reading view** — same formula, gray scrollbar visible at bottom:
<img width="1014" height="428" alt="Image" src="https://github.com/user-attachments/assets/09f55843-bf11-4a04-85f0-a8d3b8d42aae" />

## Environment

- Obsidian: v1.8.x (Windows)
- Plugin version: latest
